tests/comparisons/Template/testBakeIndex.ctp

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
   * @var \Bake\Test\App\View\AppView $this
-  * @var \Bake\Test\App\Model\Entity\TemplateTaskComment[]|\Cake\Collection\CollectionInterface $templateTaskComments
+  * @var \Bake\Test\App\Model\Entity\Cake\Datasource\EntityInterface[]|\Cake\Collection\CollectionInterface $templateTaskComments
```

Comment:
That looks a little weird, the interface name shouldn't be prefixed with `\Bake\Test\App\Model\Entity`.